backend/inspirehep/search/api.py

```diff
@@ -277,6 +277,11 @@ def get_records_by_pids(pids, source=None, size=10000):
         return results.execute().hits
 
 
+class LiteratureAggregationsSearch(LiteratureSearch):
+    def execute(self, *args, **kwargs):
+        return super(LiteratureSearch, self).execute(*args, **kwargs)
```

Comment:
```suggestion
        return super().execute(*args, **kwargs)
```

Reply:
But I need to call the execute from InspireSearch, not LiteratureSearch.

Reply:
true my bad